jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupWebServerControlServiceImpl.java

```diff
@@ -59,13 +60,19 @@ public void controlAllWebSevers(final ControlGroupWebServerRequest controlGroupW
 
     private void controlWebServers(final ControlGroupWebServerRequest controlGroupWebServerRequest, final User user, Set<WebServer> webServers) {
         for (final WebServer webServer : webServers) {
-            executorService.submit(new Callable<CommandOutput>() {
-                @Override
-                public CommandOutput call() throws Exception {
-                    final ControlWebServerRequest controlWebServerRequest = new ControlWebServerRequest(webServer.getId(), controlGroupWebServerRequest.getControlOperation());
-                    return webServerControlService.controlWebServer(controlWebServerRequest, user);
-                }
-            });
+            if ("START".equalsIgnoreCase(controlGroupWebServerRequest.getControlOperation().name()) && "STARTED".equalsIgnoreCase(webServer.getState().toStateLabel())) {
+                LOGGER.info("webServer {} already in state {}.", webServer.getName(), webServer.getState().toStateLabel());
```

Comment:
Do we really need this ? I mean applying conditions when logging ? I guess we used this for trouble shooting and you wanted to limit the logs to specific criteria but it's not a good practice in my opinion because you will miss out on some of the log data e.g. maybe someday you would need logging on the other web server name and state combinations but since you have a condition here, then you would have to change the code to see those. In addition, we can use tools to narrow down the logs that you are analyzing instead of having our application do that.